firstvoices/backend/tests/test_apis/base_api_test.py

```diff
@@ -735,7 +735,7 @@ def test_destroy_site_missing_404(self):
         assert response.status_code == 404
 
 
-class SiteContentPatchApiTestMixin:
+class SiteContentPatchApiTestMixin(WriteApiTestMixin):
```

Comment:
I think it's best to not make this inherit from WriteApiTestMixin and instead just include a comment that they need to be used together. Having WriteApiTestMixin in the inheritance tree twice could cause surprising results in some tests, and it is already used in the base test classes